lib/zendesk_apps_support/validations/requests.rb

```diff
@@ -1,57 +1,70 @@
 # frozen_string_literal: true
 
-require 'uri'
 require 'ipaddress_2'
 
 module ZendeskAppsSupport
   module Validations
     module Requests
       class << self
-        HTTP_REQUEST_METHODS = %w[GET HEAD POST PUT DELETE CONNECT OPTIONS TRACE].freeze
-
-        REQUEST_CALLS = [
```

Comment:
I agree with stripping out the JS-request-parsing stuff in favour of just checking for the presence of an internal IP or http address. It'd be nice if we could do this, but it's really hard to catch all the edge cases with just regex.

Reply:
Thank you, Sean.